README_DEV.md

```diff
@@ -929,14 +836,34 @@ home_userresources
 home_userresources_resources
 ```
 
-## 4. Monthly routines
+## 5. Monthly & Yearly routines
 
 Once a month:
 
 - Check [antivirus](#install-antivirus). In doubt, scan the whole system with `clamscan -r -i /`, but it will be long...
 - Check [server updates](#monthly-server-update), including [Docker updates](#monthly-docker-update)
 
-## 5. Tips
+Once a year:
+
+- [Renew SSL certificate](#new-instructions-for-certificate-renewal)
+
+For these types of interventions, the typical sequence of actions is the following:
+
+1. Backup: `./dump/dump_<env db>.sh` (or `make <env>_backup`)
+1. Down: `make <env>_down`
+1. Perform specific action (e.g. server updates, certificate renewal, new release, etc.)
+1. Build & up : `make <env>_buildupd`
+
+And in case of changes in the application code:
+
+1. Migrations : `make <env>_migr` (or see the 2 associated commands)
```

Comment:
This is not required as if there are modifications in the code, there is a script for the deployement and the migrations are included